test/services/DateTimeServiceSpec.scala

```diff
@@ -39,10 +39,11 @@ class DateTimeServiceSpec extends SpecBase {
 
     "return the fixed date if fixedDateTime is enabled" in {
 
-      val app     = application().configure("features.fixed-systemdate-for-tests" -> true).build()
-      val service = app.injector.instanceOf[DateTimeService]
+      val application = applicationBuilder().configure("features.fixed-systemdate-for-tests" -> true).build()
 
-      running(app) {
+      val service = application.injector.instanceOf[DateTimeService]
```

Comment:
I don't think we need this.

Reply:
We're not using the original instance on line 42 so this will be missing the config that has been set.

Reply:
My mistake, nevermind this.